app/uk/gov/hmrc/homeofficesettledstatusstubs/services/StubDataService.scala

```diff
@@ -76,8 +76,11 @@ class StubDataService @Inject() (cc: ControllerComponents) extends BackendContro
     }
 
   private def makeResponse(makeString: String): StatusCheckResult = {
-    val _ :: product :: status :: expired =
```

Comment:
scala 2.13 complain that this match is not exhaustive so I had to do it this way